pkg/api/README.md

```diff
@@ -0,0 +1,82 @@
+# OpenAPI specifications
+
+Since version 8.4, HTTP API details are [specified](https://editor.swagger.io/?url=https://raw.githubusercontent.com/grafana/grafana/main/public/api-merged.json) using OpenAPI v2. Starting from version 9.1, there is also a [OpenAPI v3 specification](https://editor.swagger.io/?url=https://raw.githubusercontent.com/grafana/grafana/main/public/openapi3.json) (generated by the v2 one using [this script](https://github.com/grafana/grafana/blob/main/scripts/openapi3/openapi3conv.go)).
+
+# OpenAPI annotations
+
+The OpenAPI v2 specification is generated automatically from the annotated Go code using [go-swagger](https://github.com/go-swagger/go-swagger). The code annotation rules can be found [here](https://goswagger.io/use/spec.html).
```

Comment:
could link text be more descriptive? re: https://grafana.com/docs/writers-toolkit/writing-guide/useful-link-text/